app/environments/brassbirmingham/brassbirmingham/envs/classes/trade_post.py

```diff
@@ -38,21 +38,21 @@ def __init__(
         self.moneyGained = moneyGained
         self.victoryPointsGained = victoryPointsGained
         self.incomeGained = incomeGained
-        self.possibleTrades = []  # list of
+        self.merchantTiles = []  # list of merchant tiles
         self.networkPoints = networkPoints
         self.canDevelop = canDevelop
         self.networks: List[RoadLocation] = []
 
     """
-    addPossibleTrade
+    addMerchantTile
     game init use only
-    trades which are possible to make, array of Building object names 'iron', 'goods', etc...
+    trades which are possible to make, array any of ['blank', 'all', 'pottery', 'cotton', 'goods']
 
-    :param possibleTrade: possibleTrade
+    :param merchantTile: merchantTile
     """
 
-    def addPossibleTrade(self, possibleTrade):
-        self.possibleTrades.append(possibleTrade)
+    def addMerchantTile(self, merchantTile: str):
+        self.merchantTiles.append(merchantTile)
```

Comment:
Do we track what beer corresponds to which merchant tile? Like if one of the tiles is "pottery" and one of the tiles is "cotton", you can't use both beers for selling pottery. Only one for each

Reply:
Not yet, I figured' that be part of the bonuses implementation

Reply:
Makes sense